internal/common/types.go

```diff
@@ -0,0 +1,212 @@
+package store
```

Comment:
Is this right? I'm not sure if this belongs to the `store` package.

Reply:
Thanks for catching this! I guess `package common` was in the commit I removed...